Dockerfiles/Dockerfile.init

```diff
@@ -0,0 +1,6 @@
+# Init container
+FROM bitnami/kubectl:1.25.6 AS init
```

Comment:
The ubi-minimal image can be used as a base image, since that will be required for operator certification. In this case, you would need to install the kubectl binary during image creation. See here for reference to the current ubi-minimal image that is being used: https://github.com/dell/csi-powerflex/blob/main/overrides.mk#L6

Reply:
We got to know about the base image requirement. @donatwork is currently working on common dell/kubectl image which we can switch to as needed.
If the init container is going to be around for few releases, I am not sure if it has to be certified. Moreover, rest of the replication images are built with scratch. When we must switch all images to use ubi-minimal as base, I can work on converting all at that time.